srcpkgs/caddy/INSTALL

```diff
@@ -1,6 +1,5 @@
 case "${ACTION}" in
 post)
-	setcap CAP_NET_BIND_SERVICE=+ep usr/bin/caddy
-	chown -R caddy:caddy etc/caddy
```

Comment:
The `chown` must be left intact, otherwise caddy isn't able to read the file on startup (you vinstall 600 the Caddyfile in template).